etc/clang-format.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.evergreen/init.sh"
+
+: "${CLANG_FORMAT_VERSION:=15.0.7}"
+export PIPX_HOME="${BUILD_CACHE_DIR}/pipx"
+
+if ! run_python -c ''; then
+  fail "No Python found?"
+fi
+
+if ! run_python -c 'import pipx'; then
```

Comment:
pipx seems to require Python 3.6 or newer. `run_python` defefined here:

https://github.com/mongodb/libmongocrypt/blob/73302cd485cd63becec92a29dff5f09f2731b4ab/.evergreen/init.sh#L222

seems to simply select `python` which defaults to Python 2 in many environments. Can Python binary selection be improved to ensure a Python 3 binary is used for pipx?

Reply:
Yes. Python selection needs to be improved in general, since the filename varies sporadically between platforms. It's very obnoxious.